### PR TITLE
feat: added liked videos & watch later pages in video library

### PR DESCRIPTION
**Pull request by kedark152**
### feat: added liked videos & watch later pages in video library

Preview: https://deploy-preview-5--agroplay.netlify.app/

Features:-
- refactor:- renamed usePlaylist hook to useVideoAction hook for adding other video actions.
- add/ remove video from watch later list in db.
- add/remove video from liked videos list in db.
- display the complete list on watch later page & liked video page from db via api calls.

### CODE REVIEW

File: src/components/VideoOptions.js [hunk 1]
```diff
@@ -1,31 +1,70 @@
...
             }
           >
             <i className="material-icons">playlist_add</i>
             Save to playlist
           </li>
-          <li>
-            <i className="material-icons">thumb_up</i>Add to liked videos
+          <li
+            onClick={() => {
+              isInLikesList(activeVideo._id)
```
Comment (hinanshisuthar) [inline src/components/VideoOptions.js]: You can destruture it before to avoid repetition

File: src/pages/Home.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ export const Home = () => {
               displayVideos.map((video) => (
                 <VideoCard key={uuid()} videoCardDetails={video} />
```
Comment (hinanshisuthar) [inline src/pages/Home.js:47]: You can pass the video id here instead of using uuid

**Review by hinanshisuthar — COMMENTED** [review]
Overall good code written suggested changes. However, optional. Nice logic implementation too. Good to merge!

File: src/components/VideoOptions.js [hunk 3]
```diff
@@ -1,31 +1,70 @@
...
+  };
+
   return (
     <div className="video-options-box flex-column">
       <RequiresAuth>
         <ul>
-          <li>
-            <i className="material-icons">watch_later</i>Save to watch later
+          <li
+            onClick={() => {
```
Comment (AnuragSahu11) [inline src/components/VideoOptions.js]: If the onClick function block becomes too big/cluttered then you can separate it into a function.

**Review by AnuragSahu11 — APPROVED** [review]
Good function names, nice error handling, you can merge

File: src/components/VideoOptions.js (hunk 3 above)
Comment (kedark152) [reply src/components/VideoOptions.js]: Thanks, will implement the same.

**Comment by kedark152** [comment]
Thanks Hinanshi & Anurag for PR Review.

**Pull request merged**